validators/structure_validator.pyi

```diff
@@ -1,5 +1,6 @@
 from dodona.translator import Translator
 
+def get_similarity(sol: str, sub: str) -> (float, float, float): ...
```

Comment:
Can you use `Typing.Tuple` for these type annotations? There's a few more throughout this PR.